lib/auth/desktop.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"crypto/x509"
 	"crypto/x509/pkix"
+	encoding_binary "encoding/binary"
```

Comment:
Why do we need to alias the import? What's wrong with just referring to it as `binary`?